docs/source/object_detection.mdx

```diff
@@ -154,6 +154,9 @@ You can verify the transform works by visualizing the 10th example:
 ... )
 ```
 
+To learn how to leverage parts of 🤗 datasets end-to-end for training an object detection model, refer to
+[this notebook](https://nbviewer.org/github/NielsRogge/Transformers-Tutorials/blob/master/YOLOS/Fine_tuning_YOLOS_for_object_detection_on_custom_dataset_%28balloon%29.ipynb).
+
```

Comment:
@NielsRogge is there a plan to leverage the `datasets` library in this notebook? I have gone through the notebook and it felt like it was necessary to leverage `torchvision` in some parts to get the data pipeline ready. 

Let me know.